package-lock.json

```diff
@@ -6781,8 +6781,7 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true,
-          "optional": true
+          "bundled": true
```

Comment:
Was there a package change? If not, please remove this file from the PR. Generally this should only update when you update a package.